CMakeLists.txt

```diff
@@ -159,20 +159,45 @@ if (WIN32)
     add_definitions(-DNOMINMAX -DWIN32_LEAN_AND_MEAN)
 endif()
 
-# Dependencies
+if (ANDROID)
+    set(CMAKE_FIND_ROOT_PATH ${OPENMW_DEPENDENCIES_DIR} "${CMAKE_FIND_ROOT_PATH}")
```

Comment:
Would it be possible to change the CMAKE_FIND_ROOT_PATH in your cmake call? e.g. instead of
<pre>
cmake -DOPENMW_DEPENDENCIES_DIR=path/to/deps
</pre>
you would call
<pre>
cmake -DCMAKE_FIND_ROOT_PATH=path/to/deps
</pre>

This way we avoid cluttering our cmake script with more platform specific code.

Also by the way, the CMAKE_FIND_ROOT_PATH is set twice, once on line 163 and once on line 49.